test/unrunnableTestCases.csv

```diff
@@ -2064,11 +1914,9 @@ cbcl-castable-dateTime-002,Error: FODT0001: Datetime year is out of bounds
 cbcl-castable-dayTimeDuration-001,AssertionError: Expected XPath "P11768614336404564651D" castable as xs:dayTimeDuration to resolve to false: expected true to be false
 cbcl-castable-dayTimeDuration-002,AssertionError: Expected XPath "-P11768614336404564651D" castable as xs:duration to resolve to false: expected true to be false
 cbcl-castable-duration-001,AssertionError: Expected XPath "-P768614336404564651Y" castable as xs:duration to resolve to false: expected true to be false
-cbcl-castable-duration-002,AssertionError: Expected XPath "P768614336404564651Y" castable as xs:duration to resolve to false: expected true to be false
-cbcl-castable-name-002,Error: XPST0017: Function fn:current-time with arity of 0 not registered. No similar functions found.
-cbcl-castable-ncname-002,Error: XPST0017: Function fn:current-time with arity of 0 not registered. No similar functions found.
+cbcl-castable-duration-002,Error: FODT0002: Value overflow while constructing xs:yearMonthDuration
 cbcl-castable-yearMonthDuration-001,AssertionError: Expected XPath "-P768614336404564651Y" castable as xs:yearMonthDuration to resolve to false: expected true to be false
-cbcl-castable-yearMonthDuration-002,AssertionError: Expected XPath "P768614336404564651Y" castable as xs:yearMonthDuration to resolve to false: expected true to be false
+cbcl-castable-yearMonthDuration-002,Error: FODT0002: Value overflow while constructing xs:yearMonthDuration
```

Comment:
The test case is 
`   <test-case name="cbcl-castable-yearMonthDuration-002">
      <description> test castable to xs:yearMonthDuration with overflow </description>
      <created by="Tim Mills" on="2008-04-25"/>
      <test>"P768614336404564651Y" castable as xs:yearMonthDuration</test>
      <result>
         <assert-false/>
      </result>
   </test-case>
`

Looks like we should return false, not throw an error?